File: 151004/Terebey/JWT/publisher/src/main/java/org/education/bean/dto/IssueResponseTo.java
package org.education.bean.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class IssueResponseTo {
    int id;
    int creatorId;
    String title;
    String content;
}

File: 151001/Hlebanova/LR1/LR1_REST/Dto/Response/IssueResponseTo.cs
using System.Text.Json.Serialization;

namespace LR1.Dto.Response;

public record IssueResponseTo(
    [property: JsonPropertyName("id")] long Id,
    [property: JsonPropertyName("creatorId")]
    long CreatorId,
    [property: JsonPropertyName("title")] string Title,
    [property: JsonPropertyName("content")]
    string Content
);